fighters/kirby/src/opff.rs

```diff
@@ -320,6 +899,260 @@ unsafe fn fastfall_specials(fighter: &mut L2CFighterCommon) {
         *FIGHTER_KIRBY_STATUS_KIND_SPECIAL_S_FALL,
         *FIGHTER_KIRBY_STATUS_KIND_SPECIAL_S_JUMP,
         *FIGHTER_KIRBY_STATUS_KIND_SPECIAL_S_ATTACK,
+        //copy abilities
```

Comment:
Instead of checking every Kirby special_n status individually, try something like this:
```rs
let status = StatusModule::status_kind(fighter.module_accessor);
if (0x206..0x37c).contains(&status) {
  // do the thing
}
```
These two values should encompass the range of the Copy Ability statuses